sklearn/linear_model/_logistic.py

```diff
@@ -1087,13 +1099,17 @@ class LogisticRegression(LinearClassifierMixin,
         Note that these weights will be multiplied with sample_weight (passed
         through the fit method) if sample_weight is specified.
 
+        |ControlParameter|
+
         .. versionadded:: 0.17
            *class_weight='balanced'*
 
     random_state : int, RandomState instance, default=None
         Used when ``solver`` == 'sag', 'saga' or 'liblinear' to shuffle the
         data. See :term:`Glossary <random_state>` for details.
 
+        |ControlParameter|
```

Comment:
This is one of those which might end up better having a completely custom type of a parameter, like `RandomSourceParameter` or something.

Reply:
`RandomSourceParameter` also illustrates the limits of such approach: I have no idea what this means.

If we try to be too precise, we will make it tedious to contribute, and will not reach any users as they won't understand.

Reply:
Maybe `random_state` should just not have any label?